apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx

```diff
@@ -114,12 +121,15 @@ export const DynamicGovernanceConnector: FunctionComponent<DynamicGovernanceConn
         }
     };
 
-    const handleSubmit = (values) => {
-        const data = {
+    const handleSubmit = (values: Record<string, string | boolean>) => {
+        const data: {
```

Comment:
Can we check if we can use `UpdateGovernanceConnectorConfigInterface` interface here?

Reply:
We can

Reply:
Fixed by https://github.com/wso2/identity-apps/compare/196e4bc1809d898712a32fd14d313d859fc739ec..8fd831d8584b09497fbab6218a7b567d712d87df